src/AppBundle/Tests/Controller/DefaultControllerTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AppBundle\Tests\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+/**
+ * Functional test that implements a "smoke test" of all the public and secure
+ * URLs of the application.
+ * See http://symfony.com/doc/current/best_practices/tests.html#functional-tests.
+ *
+ * Execute the application tests using this command (requires PHPUnit to be installed):
+ *
+ *     $ cd your-symfony-project/
+ *     $ phpunit -c app
+ *
+ */
+class DefaultControllerTest extends WebTestCase
+{
+    /**
+     * PHPUnit's data providers allow to execute the same tests repeated times
+     * using a different set of data each time.
+     * See http://symfony.com/doc/current/cookbook/form/unit_testing.html#testing-against-different-sets-of-data.
+     *
+     * @dataProvider getPublicUrls
+     */
+    public function testPublicUrls($url)
+    {
+        $client = self::createClient();
+        $client->request('GET', $url);
+
+        $this->assertTrue(
+            $client->getResponse()->isSuccessful(),
+            sprintf('The %s public URL loads correctly.', $url)
+        );
+    }
+
+    /**
```

Comment:
__[development][code_sniffer]__
* Doc comment for parameter "$url" missing

<!-- Comment by [SideCI](https://sideci.com) -->